src/codecatalyst/activation.ts

```diff
@@ -95,11 +105,20 @@ export async function activate(ctx: ExtContext): Promise<void> {
                 }
             })
         }
-    }
 
-    Commands.register('aws.codecatalyst.removeConnection', () => {
-        authProvider.removeSavedConnection()
-    })
+        const maxInactivityMinutes = thisDevenv.summary.inactivityTimeoutMinutes
+        const devEnvClient = thisDevenv.devenvClient
+        if (shouldTrackUserActivity(maxInactivityMinutes)) {
+            InactivityMessages.instance.setupMessages(
+                maxInactivityMinutes,
+                devEnvClient,
+                InactivityWarningMessage.instance,
+                InactivityShutdownMessage.instance
+            )
```

Comment:
Feels like too many singletons. Why can't `InactivityMessages` create the messages?

Reply:
Removed all singletons